DigitalLearningSolutions.Data/Services/DelegateApprovalsService.cs

```diff
@@ -128,5 +160,34 @@ string emailAddress
 
             return new Email(emailSubject, body, emailAddress);
         }
+
+        private static Email GenerateDelegateRejectionEmail(
+            string? delegateName,
+            string emailAddress)
+        {
+            string emailSubject = "Digital Learning Solutions registration rejected"; // TODO AIR-424 capitalisation inconsistent with approvals
+
+            var body = new BodyBuilder
+            {
+                TextBody = $@"Dear {delegateName},
+                        Your Digital Learning Solutions (DLS) registration at the centre [Centre Name] has been rejected by an administrator.There are several reasons that this may have happened including:

```

Comment:
Is [Centre Name] supposed to be an interpolated value? Similarly in the Html version